full-check.sh

```diff
@@ -12,8 +12,11 @@ printf "========================= MYPY STATIC TYPE CHECKER =================\n"
 mypy --config-file=mypy.ini concent_api/
 printf "\n"
 
-printf "============================ UNIT TESTS ============================\n"
+printf "========================= UNIT TESTS WITH COVERAGE =================\n"
 # NOTE: 'manage.py test' does not find all tests unless we run it from within the app directory.
+rm -rf contrib/coverage/coverage_html
 cd concent_api/
-python3 manage.py test
+coverage run --rcfile=../coverage-config --source='.' manage.py test
+coverage html --rcfile=../coverage-config
+coverage report -m
```

Comment:
Please use long form of options whenever possible. It makes the calls easier to understand for someone who is not familiar with the command and we're not running it by hand to the command does not need to be short.

So, please replace `-m` with `--show-missing`.

Reply:
:+1: